deno/payloads/v10/channel.ts

```diff
@@ -161,9 +165,14 @@ export interface APIDMChannelBase<T extends ChannelType> extends Omit<APITextBas
 	recipients?: APIUser[];
 }
 
-export type APIDMChannel = APIDMChannelBase<ChannelType.DM>;
+export interface APIDMChannel extends Omit<APIDMChannelBase<ChannelType.DM>, 'name'> {
+	/**
+	 * The name of the channel (always null for DM channels)
```

Comment:
Is it always returned as null or can it be undefined too? If it can't be undefined, drop the `?`